apparmor/fr.emersion.Mako

```diff
@@ -0,0 +1,22 @@
+#include <tunables/global>
+
+profile fr.emersion.Mako /usr/bin/mako {
+  #include <abstractions/base>
+  #include <abstractions/fonts>
+  #include <abstractions/wayland>
+
+  #include <abstractions/dbus-strict>
+  dbus bind
+       bus=session
+       name=org.freedesktop.Notifications,
+  dbus receive
+       bus=session
+       path=/fr/emersion/Mako
+       interface=fr.emersion.Mako,
+
+  /{run,dev}/shm/mako-* rw,
```

Comment:
`/run/shm` doesn't exist for me.

Reply:
Right, it is fairly niche. Could still pop up.

Reply:
Where is it coming from?

Reply:
https://github.com/zfsonlinux/mountall/blob/57619b318090a4226c69d24d035e6568ec2ca0c9/src/fstab#L23

Reply:
https://wiki.debian.org/ReleaseGoals/RunDirectory#Packages_using_.2Fdev.2Fshm

Reply:
Oh okay, so it's the thing that will replace `/dev/shm`. Let's keep it.